test/runtests.jl

```diff
@@ -11,8 +11,10 @@ ENV["GKSwstype"] = "100"
 ENV["DATADEPS_ALWAYS_ACCEPT"] = true
 
 LOG_DIRS = Any["test_logs/"]
+test_s3 = false # Minio breaks on my system
```

Comment:
should this be turned on on CI?

Reply:
As the comment suggests, this throws Minio errors on my machine which I was unable to fix, so I disabled it.
If you can run the tests with `true` here it would be a big help :+1: 
If it works for you, then sure lets turn that on for CI